.github/workflows/stale.yml

```diff
@@ -0,0 +1,81 @@
+# See https://github.com/actions/stale
+
+name: Stale issues and pull-requests
+
+on:
+  schedule:
+    # Run once a day
+    # N.B. "should" be quoted, according to
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    if: "github.repository == 'metoppv/improver'"
+    runs-on: ubuntu-latest
+    steps:
+        # Using actions/stale@v5
+      - uses: actions/stale@3cc123766321e9f15a6676375c154ccffb12a358
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Idle number of days before marking issues/prs stale.
+          days-before-stale: 60
+
+          # Idle number of days before closing stale issues/prs.
+          days-before-close: 30
```

Comment:
Ooo closes issues too?

Is this additive? So it'd be after 90 days of no activity?

Reply:
Yep

Reply:
We often write issues to capture work for the future, even if we don't intend to action it immediately. We also have chains of work that will take more than 3 months to get through, but for which we have written out the skeleton of work in ticket form. Most of this is on the blue team repo, so would not be affected by this, but in principle I am not sure we want to close issues in any way other than manually.

I'm all for the PR closure. I'd be happy with a shorter period before marking them as stale, say 30 days, then closure 30 days after that, but if that's a bit too short, I'm happy with 90 days.

Reply:
Thanks @bayliffe